container-images/http-server/index.ts

```diff
@@ -0,0 +1,40 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import { env } from '@container-images/env-inject'
+import express from 'express'
+
+const {
+  npm_package_name,
+  npm_package_version,
+  npm_package_git_commit,
+  HTTP_KEEP_ALIVE_TIMEOUT = 15000,
```

Comment:
FWIW I think a 503 is probably preferable to a 15s response time for both parties, but this lets us control that and is a nice cleanup to boot - LGTM